packages/core/admin/admin/src/content-manager/hooks/useFindRedirectionLink/utils/tests/getRedirectionLink.test.js

```diff
@@ -60,7 +60,7 @@ describe('CONTENT MANAGER | Containers | CollectionTypeFormWrapper | utils', ()
       expect(mergeParams({ ok: true }, null)).toEqual({ ok: true });
     });
 
-    it('should merge 2 complexe objects correctly', () => {
+    it('should merge 2 complex objects correctly', () => {
```

Comment:
It's possible `complexe` is a term of art...

Reply:
again, it's a test